_posts/2022-04-11-1.15-announcement.md

```diff
@@ -0,0 +1,402 @@
+sources that produce watermarks at a different speed and when the source has the same 
+parallelism as splits/shards/partitions.
+
+
+### SQL version upgrades
+
+The execution plan of SQL queries and its resulting topology is based on optimization 
+rules and a cost model. This means that even minimal changes could introduce a completely 
+different topology. This dynamism makes guaranteeing snapshot compatibility 
+very challenging across different Flink versions. In the efforts of 1.15, the community focused 
+on keeping the same query (via the same topology) up and running even after upgrades. 
+
+At the core of SQL upgrades are JSON plans 
+([please note that we only have documentation in our JavaDocs for now and are still working on updating the documentation](https://nightlies.apache.org/flink/flink-docs-release-1.15/api/java/org/apache/flink/table/api/CompiledPlan.html)), which are JSON functions that make it easier to import and export structured data in SQL. This has been introduced for 
+internal use already in previous releases and will now be exposed externally. Both the Table API 
+and SQL will provide a way to compile and execute a plan which guarantees the same 
+topology throughout different versions. This feature will be released as an experimental MVP. 
+Users who want to give it a try already can create a JSON plan that can then be used 
+to restore a Flink job based on the old operator structure. The full feature can be expected 
+in Flink 1.16.
+
+Reliable upgrades makes Flink SQL more dependable for production use cases in the long term. 
+
+
+### Changelog state backend
+
+In Flink 1.15, we introduced the MVP feature of the
+[changelog state](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/ops/state/state_backends/#enabling-changelog)
+backend, which aims at 
+making checkpoint intervals shorter with the following advantages:
+
+1. Less work on recovery: The more frequently a checkpoint is taken, the fewer events 
+   need to be re-processed after recovery.
+2. Lower latency for transactional sinks: Transactional sinks commit on checkpoints, 
+   so faster checkpoints mean more frequent commits.
+3. More predictable checkpoint intervals: Currently the length of the checkpoint mainly
+   depends on the size of the artifacts that need to be persisted on the checkpoint 
+   storage and by keeping the size small, it becomes more predictable.
+
+The changelog state backend helps achieve the above by continuously 
+persisting state changes on non-volatile storage while performing materialization 
+in the background.
+
+
+### Adaptive batch scheduler
+
+In 1.15, we introduced a new scheduler to Apache Flink: the 
+[Adaptive Batch Scheduler](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/deployment/elastic_scaling/#adaptive-batch-scheduler). 
+The new scheduler can automatically decide parallelisms of job vertices for batch jobs, 
+according to the size of data volume each vertex needs to process.
```

Comment:
I propose to add the benefits of adaptive batch scheduler, as below:

```
Major benefits of this scheduler includes:
1. Ease-of-use: Batch job users can be relieved from parallelism tuning.
2. Adaptive: Automatically tuned parallelisms can better fit consumed datasets which have a varying volume size every day.
3. Fine-grained: Parallelism of each job vertex will be tuned individually. This allows vertices of SQL batch jobs to be automatically assigned different proper parallelisms.
```

Reply:
I also think it's better to move this section to be right below section "Elastic scaling with reactive mode and the adaptive scheduler", because they are a bit "similar" to users.